github-action-example/workflow.yml

```diff
@@ -10,7 +10,7 @@ jobs:
       - name: Setup JFrog CLI
         uses: jfrog/setup-jfrog-cli@v1
         env:
-          RT_SERVER: ${{ secrets.RT_SERVER_TOKEN }}
+          JF_ARTIFACTORY_SERVER: ${{ secrets.JF_ARTIFACTORY_SERVER_TOKEN }}
```

Comment:
Following my suggested change to the README, let's rename *JF_ARTIFACTORY_SERVER_TOKEN* to be *JF_ARTIFACTORY_SERVER_1*.